src/phoneField/PhoneField.story.mdx

```diff
@@ -0,0 +1,96 @@
+import { Meta, Preview, Props, Story } from '@storybook/addon-docs/blocks'
+
+import { PhoneField } from './index'
+
+<Meta title="Forms|PhoneField" />
+
+# **PhoneField**
+
+<Preview>
+  <Story name="Default">
+    <p id="phoneFieldLabel">Accessibility label</p>
+    <PhoneField
+      id="phoneFieldDefault"
+      name="phoneFieldDefault"
+      aria-labelledby="phoneFieldLabel"
+      onChange={() => {}}
+    />
+  </Story>
+</Preview>
+
+### Custom
+
+<Preview>
+  <Story name="Custom">
+    <React.Fragment>
+      <p id="labelIdCustom">Accessibility label</p>
+      <PhoneField
+        id="phoneFieldIdCustom"
+        name="phoneFieldCustom"
+        className="additionalClass"
+        aria-labelledby="labelIdCustom"
+        selectFieldLabel="Phone country code"
+        textFieldTitle="Phone number"
+        textFieldPlaceholder="Phone number"
+        defaultRegionValue="FR"
+        defaultPhoneValue="0778674534"
+        countriesWhiteList={['FR', 'ES']}
+        customCountriesName={{
+          FR: 'fr label',
+          ES: 'es label',
+        }}
+        isInline={false}
+      />
+    </React.Fragment>
+  </Story>
+</Preview>
+
+## Specifications
+
+This field consists of the input select field and regular input field.
+
+> Phone input
+
+Phone input part should only accept digits and trigger numeric keyboards on mobile devices.
+
+> Country phone code
+
+The country code should be displayed next to a country flag when selection is made.
+On the country list the full name plus code should be visible.
+
+## Behaviour
+
+On small devices the country selector and the phone input field should get separated into two separate fields.
+
+On large devices the inputs should be displayed as merged. Still the full name should be visible.
+
+Maximum width of the name should be in proportion of 1:2 to the number.
+If the name is longer than available field it should get truncated.
+
+## Usage
+
+```jsx
+import { PhoneField } from '@blablacar/ui-library/build/phoneField'
+
+  <label id="labelId">Label</label>
```

Comment:
I think eslint will return an error on this one too. `<label>` >> `<p>` should fix.